pitaya-sharp/NPitaya/src/Metrics/CustomMetrics.cs

```diff
@@ -16,17 +16,54 @@ internal MetricSpec(string name, string help, string[] labels)
         }
     }
 
+    public enum HistogramBucketType
+    {
+        Exponential,
+        Linear
+    }
+
+    public struct HistogramBuckets
+    {
+        public readonly HistogramBucketType Kind;
```

Comment:
since you're using kind for the field name consider using `kind` for the type as well.